Source/AssetRipper.Processing/AnimatorControllers/VirtualAnimationFactory.cs

```diff
@@ -130,18 +130,28 @@ private static IChildMotion AddAndInitializeNewChild(this IBlendTree tree, Proce
 			IMotion? motion = state.CreateMotion(file, controller, childNodeIndex);
 			childMotion.Motion.SetAsset(tree.Collection, motion);
 
+			IBlendTreeNodeConstant targetNode = treeConstant.NodeArray[childNodeIndex].Data;
+			if (targetNode.IsBlendTree())
+			{
+				//BlendTree ChildMotions are not allowed to use TimeScale or Mirror
+				childMotion.TimeScale = 1;
+				childMotion.Mirror = false;
+			}
+			else
+			{
+				childMotion.TimeScale = targetNode.Duration == 0 ? 0.01f : 1/targetNode.Duration;
```

Comment:
> 0.01f

Justification?

Reply:
0.01f was a mistake of mine, fixed it now